index.html

```diff
@@ -29,15 +29,182 @@ <h1>
-                        <li><a href=""><img src="images/Linkedin icon.png" alt="Linked in"></a></li> 
-                        <li><a href=""><img src="images/msg.png" alt="message"></a></li>
-                        <li><a href=""><img src="images/github.png" alt="github"></a></li>
-                        <li><a href=""><img src="images/victory.png" alt="victory"></a></li>                       
+                        <li><a href="https://twitter.com/AbuTalha8T"><img src="images/Vector.png" alt="Twitter"></a></li>
+                        <li><a href="https://www.linkedin.com/in/abu-talha-8203b252/"><img src="images/Linkedin icon.png" alt="Linked in"></a></li> 
+                        <li><a href="https://messages.google.com/"><img src="images/msg.png" alt="message"></a></li>
+                        <li><a href="https://github.com/AbuTalha3"><img src="images/github.png" alt="github"></a></li>
+                        <li><a href="https://venture.angellist.com/v/najeeb-akhun/i/najeeb-akhun/portfolio/dashboard"><img src="images/victory.png" alt="victory"></a></li>                       
                     </ul>
                 </div>
             </div>
         </section>
-     </main>    
+        <section class="cards">
+            <div class="card">
+                <div class="card-image">
+                 <img src="images/card1.png" alt="">   
+                </div>
+                <div class="card-text">
+                    <h3>Tonic</h3>
+                    <div class="expertise">
+                        <ul>
+                            <li>CANOPY</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="programes">
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </div>
+                    <button>See project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-image">
+                 <img src="images/card2.png" alt="">   
+                </div>
+                <div class="card-text">
+                    <h3>Multi-Post <span>Stories</span></h3>
+                    <div class="expertise">
+                        <ul>
+                            <li>CANOPY</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
```

Comment:
- [ ] [OPTIONAL] Kindly fix your design to match the Figma template. I **HIGHLY** advise you to implement those changes, because it would increase your chance of landing a great job since **recruiters** will check your work. 👓 Having a great design would always be a plus 💯 

![Screenshot from 2023-06-03 23-22-29](https://github.com/AbuTalha3/Portfolio-Setup/assets/10439283/658ecae2-2d77-43e2-b11f-c78b68111ec1)

<h2>Take a look at the spacing after your button ☝🏽 ☝🏽 </h2>